python/ray/ml/batch_predictor.py

```diff
@@ -5,7 +5,7 @@
 from ray.ml.predictor import Predictor
 
 
-class BatchPredictor(Predictor):
+class BatchPredictor:
```

Comment:
`BatchPredictor` does not fit the `Predictor` interface. It should not subclass `Predictor`.